packages/shared-state/files/usr/bin/shared-state

```diff
@@ -83,6 +84,15 @@ function sharedState.sync(urls)
 	urls = urls or {}
 
 	if #urls < 1 then
+
+		local uci_cursor = uci:cursor()
+		local fixed_candidates = uci_cursor:get("shared-state", "options","candidates") or {}
+		for _, line in pairs(fixed_candidates) do
+			table.insert(
```

Comment:
could it be that this gets inserted every time that is called, and then the urls in the file get piled up?
first time it runs it is 1 time, 2nd time 2 times, and on and on?

Reply:
looks like today i am a little bit distracted :)
thx gio